rustup-init.sh

```diff
@@ -22,28 +22,28 @@ set -u
 # If RUSTUP_UPDATE_ROOT is unset or empty, default it.
 RUSTUP_UPDATE_ROOT="${RUSTUP_UPDATE_ROOT:-https://static.rust-lang.org/rustup}"
 
-#XXX: If you change anything here, please make the same changes in setup_mode.rs
+# NOTICE: If you change anything here, please make the same changes in setup_mode.rs
 usage() {
-    cat 1>&2 <<EOF
+    cat <<EOF
 rustup-init 1.25.2 (8c4dad73d 2023-02-01)
 The installer for rustup
 
 USAGE:
     rustup-init [FLAGS] [OPTIONS]
 
 FLAGS:
-    -v, --verbose           Enable verbose output
-    -q, --quiet             Disable progress output
-    -y                      Disable confirmation prompt.
-        --no-modify-path    Don't configure the PATH environment variable
-    -h, --help              Prints help information
-    -V, --version           Prints version information
+    -v, --verbose                        Enable verbose output
+    -q, --quiet                          Disable progress output
+    -y                                   Disable confirmation prompt.
+        --no-update-default-toolchain    Don't update any existing default toolchain after install
+        --no-modify-path                 Don't configure the PATH environment variable
+    -h, --help                           Prints help information
+    -V, --version                        Prints version information
 
 OPTIONS:
         --default-host <default-host>              Choose a default host triple
         --default-toolchain <default-toolchain>    Choose a default toolchain to install
-        --default-toolchain none                   Do not install any toolchains
```

Comment:
So this is interesting. I think we lose something by not explicitly saying 'use none to get no toolchain installed'

Reply:
I'd rather we either:
- keep them different
- teach the binary rustup-init how to offer that tip

we added that because folk were not realising that this was how it worked.

Reply:
Addressed. I also think it's rarely used by users.